### ISSUE TITLE
Port Sparrow to mac os x

### ISSUE THREAD

**Issue by IvanUshakov** [PrimaryFeather/Sparrow-Framework#1048]
Hello are you planning to port Sparrow-Framework to mac os x in the future?
Sory, for my bad English.

**Comment by PrimaryFeather** [comment]
Yes, it is planned to do so -- but I can't give you a good estimate right now. The plan is to create a Mac version of Sparrow 2.0, on which development will start soon. 

In the meantime, I can only forward you to our second game framework: Starling. It is Flash based (ActionScript 3) and has the same architecture as Sparrow. An AIR application developed with Starling will work on Windows and OS X.

http://www.starling-framework.org

**Comment by yderidde** [comment]
Any plan to port it to OSX ? or this was cancelled ?

**Comment by PrimaryFeather** [comment]
Not for this version of Sparrow (1.x), but for the 2.x fork. More information is available [here](https://github.com/Gamua/Sparrow-Framework/issues/27).
